src/it/java/io/fabric8/elasticsearch/KibanaIndexModeSharedOpsIntegrationTest.java

```diff
@@ -37,12 +38,12 @@ public void testNonAdminUserRetrievingDocuments() throws Exception {
         final String userName = "nonadminuser";
         //artificially seed user's kibanaIndex
         String kibanaIndex = getKibanaIndex(KibanaIndexMode.SHARED_OPS, userName, false);
-        givenDocumentIsIndexed(kibanaIndex, "config", "0", "myKibanaIndex");
+        givenDocumentIsIndexed(kibanaIndex, "config", "4.6.4", "myKibanaIndex");
```

Comment:
instead of hardcoding `4.6.4` everywhere can we move it into a test env var?

Reply:
you mean like an actual environment variable or just a constant?  As long as what we seed is semantically older then current it wont make a difference